cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -146,63 +146,45 @@ class OutlinePlugin: CDVPlugin {
 
   func initializeErrorReporting(_ command: CDVInvokedUrlCommand) {
     DDLogInfo("initializeErrorReporting")
-    guard let apiKey = command.argument(at: 0) as? String else {
+    guard let sentryDsn = command.argument(at: 0) as? String else {
       return sendError("Missing error reporting API key.", callbackId: command.callbackId)
     }
-    do {
-      Client.shared = try Client(dsn: apiKey)
-      try Client.shared?.startCrashHandler()
-      Client.shared?.breadcrumbs.maxBreadcrumbs = OutlinePlugin.kMaxBreadcrumbs;
-      sendSuccess(true, callbackId: command.callbackId)
-    } catch let error {
-      sendError("Failed to init error reporting: \(error)", callbackId: command.callbackId)
+    SentrySDK.start { options in
```

Comment:
I take it there's no longer a risk of non-initialization? SentrySDK.start can't fail?